python_modules/dagster-ext/dagster_ext/__init__.py

```diff
@@ -411,6 +411,17 @@ def _upload_loop(self, is_task_complete: Event) -> None:
             time.sleep(1)
 
 
+class ExtBufferedFilesystemMessageWriterChannel(ExtBlobStoreMessageWriterChannel):
+    def __init__(self, path: str, *, interval: float = 10):
+        super().__init__(interval=interval)
+        self._path = path
+
+    def upload_messages_chunk(self, payload: IO, index: int) -> None:
+        message_path = os.path.join(self._path, f"{index}.json")
+        with open(message_path, "w") as f:
+            f.write(payload.read())
+
+
```

Comment:
dbfs exposes a traditional I/O interface in python?

Reply:
yes it is mounted on `/dbfs`